modules/text_server_adv/SCsub

```diff
@@ -503,7 +503,7 @@ if env["builtin_icu"]:
 
 module_obj = []
 
-if env_text_server_adv["tools"]:
+if env_text_server_adv["tools"] and env["builtin_icu"]:
     env_text_server_adv.Append(CXXFLAGS=["-DICU_STATIC_DATA"])
 
 env_text_server_adv.Append(
```

Comment:
This and the mdsfgen (btw `msdngen_enabled` has a typo) and freetype/graphite includes should likely all be guarded with the respective `builtin_*` checks, as we don't want to use builtin headers by mistake while linking against system libraries.

Reply:
Probably, this might be an issue if system version is different.